demos/http3/ossl-nghttp3.c

```diff
@@ -613,6 +614,12 @@ int OSSL_DEMO_H3_CONN_handle_events(OSSL_DEMO_H3_CONN *conn)
         if (ec == 0)
             break;
 
+        /*
+	 * we let SSL_write_ex2(3ossl) to conclude the stream for us (send FIN)
```

Comment:
Nits: `We` and remove the `3ossl` or replace with `3` as I think the ossl is distibution-patch-specific thing.